### PR TITLE
RES-831: Add option to use "torch.optim.lr_scheduler"

### PR DESCRIPTION
**Pull request by lscheinkman**
Replaced the code updating the learning rate with torch.optim.lr_scheduler classes.
Use the `lr_scheduler` and `lr_scheduler_params` to configure the class. See  [experiments.cfg](https://github.com/numenta/htmresearch/compare/master...lscheinkman:RES-831?expand=1#diff-ac61be1fe0ccba165f37f5c3fad5f468R34)

### CODE REVIEW

File: projects/sdr_paper/pytorch_experiments/experiments.cfg [hunk 1]
```diff
@@ -31,6 +31,12 @@ datadir = "data"
 
 optimizer = SGD
 
+; Learning Rate Scheduler. See "torch.optim.lr_scheduler" for valid class names
+lr_scheduler = "StepLR"
+
+; Configure lr_scheduler class constructor using kwargs style dictionary
+lr_scheduler_params = "{'step_size': 1, 'gamma':%(learning_rate_factor)s}"
```
[the remark this answers is not in the document]
Comment (lscheinkman) [reply projects/sdr_paper/pytorch_experiments/experiments.cfg]: I think we should keep because it is easier to create experiments list on the learning_rate_factor.  See `experiments_cnn.cfg` experiment [cnn6] and [cnn10]

[the remark this answers is not in the document]
Comment (subutai) [reply projects/sdr_paper/pytorch_experiments/experiments.cfg]: Ah got it. 👍
